config/protocol.unit_testnet.single.yml

```diff
@@ -8,7 +8,7 @@ ProtocolConfiguration:
   ValidatorsCount: 1
   VerifyBlocks: true
   VerifyTransactions: true
-  P2PSigExtensions: false
+  P2PSigExtensions: true
```

Comment:
Do we really need it?

Reply:
Notary is a nice example of the contract which has `onNEP17Payment` and is able to handle `data`. However, we may create another contract with `onNEP17Payment` for this test.

Reply:
It's not that I'm against this change, I'd just like to ensure we really need it. If going without it would require creating another test contract then we're better reusing notary.